scripts/training/train_metal.py

```diff
@@ -2,27 +2,28 @@
 import os
 
 import mlflow
-from sklearn.metrics import accuracy_score, auc, confusion_matrix, f1_score, roc_curve
 from sklearn.model_selection import train_test_split
 from sklearn.preprocessing import LabelEncoder
 
 from nmrcraft.analysis.plotting import plot_confusion_matrix, plot_roc_curve
-
-# Assuming these are your custom modules
 from nmrcraft.data.dataset import load_data
-from nmrcraft.models.hyperparameter_tune import hyperparameter_tune
+from nmrcraft.evaluation.evaluation import model_evaluation
+from nmrcraft.models.model_configs import model_configs
 from nmrcraft.models.models import load_model
+from nmrcraft.training.hyperparameter_tune import HyperparameterTuner
 from nmrcraft.utils.set_seed import set_seed
 
 set_seed()
 
 
 def main(dataset_size, target, model_name):
-    """Train a model with MLflow tracking."""
-    mlflow.set_experiment("Model_Classification_Experiment")
+    # TODO: better experiment naming
+    mlflow.set_experiment("Ceci_nest_pas_un_experiment")
```

Comment:
Loving the title

Reply:
Hahaha thanks we can def work on that tho